index.html

```diff
@@ -2,53 +2,26 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Nelson Mandela Tribute Page</title>
     <link rel="stylesheet" href="css/style.css">
-    <style>
-      html{
-        font-size: 20px;
-      }
-      body{
-        font-family: 'Times New Roman', Times, serif;
-        font-weight: bold;
-      }
-      .center{
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
-        width: 50%;
-      }
-      .nel{
-        display: block;
-        margin-left: auto;
-        margin-right: auto;
-        text-align: center;
-      }
-      .des{
-        font-size: medium;
-        font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
-        font-weight: lighter;text-align: center;
-
-      }
-    </style>
   </head>
 
   <body>
-    <main id= "main">
+    <main id="main">
       <h1 id="title" class="nel">Dr. Nelson Mandela</h1>
       <p class="des">A man with honor</p> 
```

Comment:
Hello, kindly ensure you use the template's design, spacing and color as indicated in the instructions.
For instance, I'd suggest you center this section to achieve the required result.